jax/_src/numpy/fft.py

```diff
@@ -109,10 +109,77 @@ def _fft_core(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return transformed
 
 
-@implements(np.fft.fftn)
 def fftn(a: ArrayLike, s: Shape | None = None,
          axes: Sequence[int] | None = None,
          norm: str | None = None) -> Array:
+  r"""Compute discrete Fourier transform of multidimensional array along given axes.
+
+  JAX implementation of :func:`numpy.fft.fftn`.
+
+  Args:
+    a: input array
+    s: sequence of integers. Specifies the shape of the result. If not specified,
+      it will default to the shape of ``a`` along the specified ``axes``.
+    axes: integer or sequence of integers, default=None. Specifies the axes along
+      which the transform is computed.
```

Comment:
Sorry, I should have caught this before, but `axes` also needs to always be a sequence as far as I can tell.

Reply:
Done